src/Common/src/System/StringNormalizationExtensions.Unix.cs

```diff
@@ -10,34 +10,34 @@ namespace System
     static partial class StringNormalizationExtensions
     {
         [SecurityCritical]
-        public static bool IsNormalized(this string value, NormalizationForm normalizationForm)
+        public static bool IsNormalized(this string strInput, NormalizationForm normalizationForm)
```

Comment:
right. I am renaming the parameters to match the desktop so we'll be consistent.